src/test/java/org/jabref/logic/importer/fetcher/ISIDOREFetcherTest.java

```diff
@@ -0,0 +1,129 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@FetcherTest
+public class ISIDOREFetcherTest {
+
+    private ISIDOREFetcher fetcher;
+
+    @BeforeEach
+    public void setup() {
+        this.fetcher = new ISIDOREFetcher();
+    }
+
+    @Test
+    public void checkArticle() throws FetcherException {
+        BibEntry expected = new BibEntry(StandardEntryType.Article)
+                .withField(StandardField.TITLE, "Investigating day-to-day variability of transit usage on a multimonth scale with smart card data. A case study in Lyon")
+                .withField(StandardField.AUTHOR, "Oscar Egu and Patrick Bonnel")
+                .withField(StandardField.YEAR, "2020")
+                .withField(StandardField.JOURNAL, "Travel Behaviour and Society")
+                .withField(StandardField.PUBLISHER, "HAL CCSD, Elsevier")
+                .withField(StandardField.ABSTRACT, "To examine the variability of travel behaviour over time, transportation researchers need to collect longitudinal data. The first studies around day-to-day variability of travel behaviour were based on surveys. Those studies have shown that there is considerable variation in individual travel behaviour. They have also discussed the implications of this variability in terms of modelling, policy evaluation or marketing. Recently, the multiplication of big data has led to an explosion in the number of studies about travel behaviour. This is because those new data sources collect lots of data, about lots of people over long periods. In the field of public transit, smart card data is one of those big data sources. They have been used by various authors to conduct longitudinal analyses of transit usage behaviour. However, researchers working with smart card data mostly rely on clustering techniques to measure variability, and they often use conceptual framework different from those of transportation researchers familiar with traditional data sources. In particular, there is no study based on smart card data that explicitly measure day-to-day intrapersonal variability of transit usage. Therefore, the purpose of this investigation is to address this gap. To do this, a clustering method and a similarity metric are combined to explore simultaneously interpersonal and intrapersonal variability of transit usage. The application is done with a rich dataset covering a 6 months period (181 days) and it contributes to the growing literature on smart card data. Results of this research confirm previous works based on survey data and show that there is no one size fits all approach to the problem of day-to-day variability of transit usage. They also prove that combining clustering algorithm with day-to-day intrapersonal similarity metric is a valuable tool to mine smart card data. The findings of this study can help in identifying new passenger segmentation and in tailoring information and services.")
```

Comment:
Is the abstract copyrighted? Normally, we remove the abstract from the tests as most text is copyrighted. And JabRef has not the right to distribute these texts.